smart-contracts/contracts/merkle-incentives/.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2.1
```

Comment:
is this needed

Reply:
as in do we use circleci for anything?

Reply:
I suppose its something created by beaker tool. We are not using it but we could i suppose.